test/e2e/delete_test.go

```diff
@@ -250,6 +260,69 @@ func (tc *testContext) testWindowsNodeDeletion(t *testing.T) {
 	require.NoError(t, err, "could not delete test namespace")
 }
 
+// deployEmptyDirVolume deploys pods in a sleep loop across all Windows nodes
+func (tc *testContext) deployEmptyDirVolume() (*apps.Deployment, error) {
+	replicaCount := gc.numberOfBYOHNodes + gc.numberOfMachineNodes
+	dp := apps.Deployment{
+		ObjectMeta: meta.ObjectMeta{
+			Name:   "windows-pod",
+			Labels: map[string]string{"app": "windows-pod"},
+		},
+		Spec: apps.DeploymentSpec{
+			Replicas: &replicaCount,
+			Selector: &meta.LabelSelector{
+				MatchLabels: map[string]string{
+					"app": "windows-pod",
+				},
+			},
+			Template: core.PodTemplateSpec{
+				ObjectMeta: meta.ObjectMeta{
+					Name:   "windows-pod",
+					Labels: map[string]string{"app": "windows-pod"},
+				},
+				Spec: core.PodSpec{
+					OS:          &core.PodOS{Name: core.Windows},
+					Tolerations: []core.Toleration{{Key: "os", Value: "Windows", Effect: "NoSchedule"}},
+					Containers: []core.Container{{
+						Name:            "windows-container",
+						Image:           tc.getWindowsServerContainerImage(),
+						ImagePullPolicy: core.PullIfNotPresent,
+						Command:         []string{powerShellExe, "-command", "while ($true) {Start-Sleep -Seconds 1}"},
+						VolumeMounts:    []core.VolumeMount{{Name: "test-volume", MountPath: "/test/"}}},
+					},
+					Volumes: []core.Volume{{Name: "test-volume", VolumeSource: core.VolumeSource{EmptyDir: &core.EmptyDirVolumeSource{
+						Medium: core.StorageMediumDefault,
+					}}}},
+					NodeSelector: map[string]string{"kubernetes.io/os": "windows"},
+				},
+			},
+		},
+	}
+	created, err := tc.client.K8s.AppsV1().Deployments(tc.workloadNamespace).Create(context.TODO(), &dp, meta.CreateOptions{})
+	if err != nil {
+		return nil, errors.Wrapf(err, "error creating deployment %v", dp)
+	}
+	return created, nil
+}
+
+// waitUntilEmptyDirScaled will return nil if the deployment is fully deployed across the Windows nodes
+func (tc *testContext) waitUntilEmptyDirScaled(name string, desiredReplicas int) error {
```

Comment:
This function looks to be the same as https://github.com/openshift/windows-machine-config-operator/blob/d775f625191d16730f8e86d42f98a9ebbe921753/test/e2e/network_test.go#L568C1-L569
please use that instead if possible